npm/package.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "@electron/get": "^1.0.1",
     "@types/node": "^14.6.2",
-    "extract-zip": "^1.0.3"
+    "extract-zip": "^1.4.1"
```

Comment:
extract-zip@latest is 2.0.1. @malept is there any reason we shouldn't update to extract-zip@latest?

Reply:
`extract-zip@^2.0.0` requires Node 10, if we want to do that for Electron 12.

Reply:
Hm, seems friendlier to support more node versions if possible. In that case, we should perhaps go for extract-zip@1.7.0

Reply:
:man_shrugging: I don't know what sort of guarantees the Electron NPM module itself has, but Ecosystem WG modules generally only support Node versions that the Node.js folks actually support.

I'd also like to point out that extract-zip v2 has fewer transitive dependencies (which is why it requires a specific minor version of Node 10).

Reply:
good q. @electron/wg-ecosystem could maybe weigh in here.

Reply:
I think Releases/Upgrades owns the `electron` NPM module, I was just pointing out what Ecosystem WG's similar policy has been.

Reply:
I don't have any context personally, and from what I can tell, our minimum Node version was last explicitly upgraded to Node 8 in Electron 7 so we could use `@electron/get` (ref https://github.com/electron/electron/pull/18413).

It makes sense to me to bump the minimum version of Node, though, especially since Node 8 reached EOL in 2019. Where do we document our minimum Node version for `electron`? We don't have an `engines` property in the `package.json`.

Reply:
> Where do we document our minimum Node version for `electron`? We don't have an `engines` property in the `package.json`.

https://github.com/electron/electron/blob/434f7749db9d3881695740d2a18fb1f4e652d0cc/npm/package.json#L15-L17